drivers/hmis/app/graphql/mutations/update_form_definition.rb

```diff
@@ -12,13 +12,14 @@ class UpdateFormDefinition < CleanBaseMutation
     field :form_definition, Types::Forms::FormDefinition, null: true
 
     def resolve(id:, input:)
-      raise 'not allowed' unless current_user.can_manage_forms?
+      access_denied! unless current_user.can_manage_forms?
 
       definition = Hmis::Form::Definition.find_by(id: id)
       raise 'not found' unless definition
       raise 'not allowed to change identifier' if input.identifier.present? && input.identifier != definition.identifier
 
       definition.assign_attributes(**input.to_attributes)
+      definition.definition = convert_form_definition(JSON.parse(input.definition))
```

Comment:
I don't think we need the extra `convert_form_definition` function, since it's doing the same thing. This should suffice:
```suggestion
      definition.definition = convert_form_item(JSON.parse(input.definition))
```